types/v56/types.go

```diff
@@ -1740,44 +1740,24 @@ type LBAppRule struct {
 
 type LBAppRules []LBAppRule
 
-// LoadBalancerVirtualServer represents a load balancer virtual server.
```

Comment:
Why we don't these types? didn't you add with previous PR's?

Reply:
Nope. These were old leftover from sometime ago. They aren't the same.